third_party/terraform/website/docs/d/service_account_key.html.markdown

```diff
@@ -33,17 +32,17 @@ data "google_service_account_key" "mykey" {
 
 The following arguments are supported:
 
-* `name` - (Required) The name of the service account key. This must have format
-    `projects/{PROJECT_ID}/serviceAccounts/{ACCOUNT}/keys/{KEYID}`, where `{ACCOUNT}`
-    is the email address or unique id of the service account.
+- `name` - (Required) The name of the service account key. This must have format
```

Comment:
While pretty much harmless once compiled, using `-` here instead of `*` just makes the formatting inconsistent with the rest of the our documentation. Do you mind switching them all back?

Reply:
Not at all! I'll update that 👍🏼